packages/ui-scripts/lib/utils/git.js

```diff
@@ -130,7 +130,7 @@ exports.commitVersionBump = function commitVersionBump(releaseVersion) {
 }
 
 exports.tagCommit = function tagCommit(releaseVersion) {
-  runGitCommand(['tag', `v${releaseVersion}`])
+  runGitCommand(['tag', '-a', `v${releaseVersion}`, '-m', `v${releaseVersion}`])
```

Comment:
I'd add a comment here why we are using the -a and -m flags. Its not trivial that omitting these breaks Lerna :)